package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@zowe/zos-ftp",
-  "version": "0.1.0",
+  "name": "@brightside/zos-ftp-for-zowe-cli",
```

Comment:
I believe we can keep the name with an `@zowe/` scope since this is a contributed plugin. : )

Reply:
I changed it to `@zowe/zos-ftp-for-zowe-cli` now. :-)